README.md

```diff
@@ -76,6 +76,15 @@ If the version is not recognized, `kubectl-validate` will attempt to look up
 the schemas for the selected version in the official upstream Kubernetes repository
 on GitHub.
 
+## overlay-schemas
+
+The ```--overlay-schemas``` option enables users to apply JSON patches onto resource schemas before they undergo validation. This feature allows for customizing the behavior of schema validation according to specific requirements.
```

Comment:
@alexzielenski I'll let you review the line, you know the usage better than I do.